src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -522,32 +522,50 @@ export class NativeHostMainService extends Disposable implements INativeHostMain
 			return shell.openExternal(url);
 		}
 
+		let browserName = configuredBrowser;
+		if (configuredBrowser.toLowerCase() === 'edge') {
+			browserName = isWindows ? 'msedge' : 'microsoft-edge';
+		}
+
 		if (configuredBrowser.includes(posix.sep) || configuredBrowser.includes(win32.sep)) {
 			const browserPathExists = await Promises.exists(configuredBrowser);
 			if (!browserPathExists) {
 				this.logService.error(`Configured external browser path does not exist: ${configuredBrowser}`);
-				return shell.openExternal(url);
+				return;
 			}
 		}
 
 		try {
 			const { default: open } = await import('open');
 			const res = await open(url, {
 				app: {
-					// Use `open.apps` helper to allow cross-platform browser
-					// aliases to be looked up properly. Fallback to the
-					// configured value if not found.
-					name: Object.hasOwn(open.apps, configuredBrowser) ? open.apps[(configuredBrowser as keyof typeof open['apps'])] : configuredBrowser
-				}
+					name: browserName
```

Comment:
I think we still need to lookup the browser name per platform using `open.apps` helper, no?